External/FEXCore/Source/Interface/Core/GdbServer.cpp

```diff
@@ -984,7 +985,7 @@ GdbServer::HandledPacketType GdbServer::ThreadAction(char action, uint32_t tid)
     case 's': {
       CTX->Step();
       SendPacketPair({"OK", HandledPacketType::TYPE_ACK});
-      auto str = fmt::format("T05thread:{:02x};", getpid());
+      fextl::string str = fmt::format("T05thread:{:02x};", getpid()).c_str();
```

Comment:
Not sure I pointed this out before, but should be `fextl::fmt::format` (feel free to skip this one if that's done in the followup PR though).